targets/switchboard/artillery-scripts/send-message.yaml

```diff
@@ -0,0 +1,106 @@
+config:
+  environments:
+    staging:
+      target: "https://staging.switchboard.assemble.live"
+      phases:
```

Comment:
Question: Do we want to treat these like unit tests where these tests all need to pass before a production release occurs?

If failing tests are ok, considering you saw good performance up to 500 so far, you could add tests up to 1000? Our Bandwidth rate limit for the registered account is now set to 900, so that feels like a goal to shoot for even if we're not ready to get there yet.

Reply:
I like the idea of testing on a kind of log scale beyond our current external rate limits so we know our sending rate is artificially limited by policy rather than software performance.

So like Aashish mentioned I would set phases for like 100, 250, 500, 1000, 2000, 5000 to start with and then adjust our passing expectations accordingly so we have a good sense of "max possible performance" within a hundred or so.